config/config.go

```diff
@@ -60,28 +62,32 @@ func bindFlags(flagSet *pflag.FlagSet) error {
 	return nil
 }
 
-// Resolve ethereum network based on the set boolean flags.
-func (c *Config) resolveEthereumNetwork(flagSet *pflag.FlagSet) error {
+// Resolve Ethereum and Bitcoin networks based on the set boolean flags.
+func (c *Config) resolveChainsNetworks(flagSet *pflag.FlagSet) error {
 	var err error
 
-	c.Ethereum.Network, err = func() (commonEthereum.Network, error) {
+	c.Ethereum.Network, c.Bitcoin.Network, err = func() (
+		commonEthereum.Network,
+		bitcoin.Network,
+		error,
+	) {
 		isGoerli, err := flagSet.GetBool(commonEthereum.Goerli.String())
 		if err != nil {
-			return commonEthereum.Unknown, err
+			return commonEthereum.Unknown, bitcoin.Unknown, err
 		}
 		if isGoerli {
-			return commonEthereum.Goerli, nil
+			return commonEthereum.Goerli, bitcoin.Testnet, nil
 		}
 
 		isDeveloper, err := flagSet.GetBool(commonEthereum.Developer.String())
 		if err != nil {
-			return commonEthereum.Unknown, err
+			return commonEthereum.Unknown, bitcoin.Unknown, err
 		}
 		if isDeveloper {
-			return commonEthereum.Developer, nil
+			return commonEthereum.Developer, bitcoin.Regtest, nil
 		}
 
-		return commonEthereum.Mainnet, nil
+		return commonEthereum.Mainnet, bitcoin.Mainnet, nil
```

Comment:
> We should rename --goerli flag to --testnet

We could but certainly not this PR because this is a broad change that impacts client deployment. On the other hand, I think that it makes sense to have explicit Ethereum networks instead of a general `--testnet` flag. If we want to add, for example, Sepolia, we will add another flag in parallel so we can have two Ethereum testnets independently.

> The fact --goerli dictates Bitcoin network used is confusing.

Why? It makes sense to use Bitcoin testnet with Ethereum testnets, Bitcoin mainnet with Ethereum mainnet etc. The only exception that makes sense is using Bitcoin testnet with Ethereum local chain, but then we can always explicitly set the `bitcoin.ethereum` section.

Reply:
As someone not familiar with this PR - say another client developer or user - I would be confused about `--goerli` flag.  This flag clearly connects the client to Goerli Ethereum testnet, but how do I select Bitcoin testnet? 

Regarding `--sepolia` and `--goerli`, the argument makes sense but in reality, we will support only one testnet network. 

This is not the current state, but ultimately, I'd like to have one client binary where I select `--testnet` for Sepolia (will replace Goerli) + Bitcoin testnet, and `--developer` for local Ethereum and regtest. Mainnet Ethereum + mainnet Bitcoin would be the default one. Renaming `--goerli` to `--testnet` is the first step. The next step is figuring out the build process (out of scope currently) so that we build one client binary instead of testnet-specific and mainnet-specific versions.

Reply:
Ok makes sense. Added a TODO c31590d5

Reply:
Captured this work here: https://github.com/keep-network/keep-core/issues/3579